app/full_text_search.py

```diff
@@ -106,16 +109,78 @@ def search(repo, query_txt, max_results):
             results.append(result)
     return results
 
-def create_document(**kwargs):
+
+def create_jp_name_document(**kwargs):
+    """
+    Creates document(romanized jp name data) for full text search.
+    It should be called in create_document method.
+    """
+    fields = []
+    romanized_names_list = []
+    for field in kwargs:
+        romanized_japanese_name =\
+                script_variant.romanize_japanese_name_by_name_dict(kwargs[field])
+        romanized_names_list.append(romanized_japanese_name)
+        fields.append(
+            appengine_search.TextField(name=field+'_romanized_by_jp_name_dict',
+                                       value=romanized_japanese_name)
+        )
+    romanized_jp_names =\
```

Comment:
How is the colon-separated name list used in indexing and search? It was not immediately clear to me. Maybe a short comment would help.